### ISSUE TITLE
Laravel 5.0 Support

### ISSUE THREAD

**Issue by stevebauman** [fojuth/plupload#9]
Any thought's on updating this package for 5.0? I believe most from 5.0 is standard as it is from 4.2, however, you'll have to change the blade syntax due to taylor changing the blade tags from `{{ }}` to `{!! !!}`.

**Comment by svpernova09** [comment]
The 2 pending 4.2 PRs would be a good start to testing Laravel 5. Also, 5 is ~ a month out. @fojuth has stated he may not update this. I've been maintaining a fork because I've needed to upgrade to 4.2 and tweak some TwigBridge related stuff

**Comment by stevebauman** [comment]
Is your fork available through composer? If this package is left in the dust and @fojuth hasn't accepted any pulls recently, I'd much rather submit pulls to a fork that has a dev available to accept them.

**Comment by svpernova09** [comment]
You can add this section to your composer.json

```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/RocketFuel-Memphis/plupload.git"
        }
    ],
```

That will tell composer to fetch the different fork instead of what packagist reports.

I've not tested this fork with Blade, so if you run into issues let me know

**Comment by fojuth** [comment]
Guys, sorry for the delay. I've been on vacation and got back a few days ago. I want to focus on making plupload up and running for laravel 5. I will revise the pull requests (I think one of them won't work like @svpernova09 wrote, but will check it). Then I'll tag the changes and create a new version for laravel 5. Once again, sorry for the delay. I'll get to it ASAP.

**Comment by svpernova09** [comment]
@fojuth good news! thanks!

**Comment by stevebauman** [comment]
@fojuth Hey that's seriously no problem at all, there's no rush here. This package is free and I don't expect someone to use their free time to manage it. I was just curious if this was in the works at some point or another. I really appreciate all the hard work and will definitely be waiting patiently for an update :D

**Comment by fojuth** [comment]
Released version 1.3, compatible with Laravel 5. Tested on Twig and Blade - both work fine. Should you run into any issues, let me know. Cheers!
